### PR TITLE
feat: add quickfix keybind

### PR DESCRIPTION
**Pull request by wyrid**
Currently there is no binding for opening the output in the quickfix window, you have to navigate to the actions menu and select the quickfix option. This will add a binding (default <C-q>) to open the quickfix window from the task list.

### CODE REVIEW

File: lua/overseer/task_list/bindings.lua [hunk 1]
```diff
@@ -50,6 +50,13 @@ M = {
       sidebar:run_action("open float")
     end,
   },
+  {
+    desc = "Open task terminal in a quickfix window",
```
Comment (stevearc) [inline lua/overseer/task_list/bindings.lua:54]: This isn't quite accurate. Could we change it to something like "open task output in quickfix window?"

**Review by stevearc — CHANGES_REQUESTED** [review]
Thanks for the PR! Looks good, just one small request about the wording

File: lua/overseer/task_list/bindings.lua (hunk 1 above)
Comment (wyrid) [reply lua/overseer/task_list/bindings.lua:54]: Whoops I didn't notice that. I will fix it and push the change.

**Review by stevearc — APPROVED** [review]
Excellent, thank you!

**Pull request merged**